doc/api/process.markdown

```diff
@@ -238,8 +238,8 @@ A `Writable Stream` to `stdout` (on fd `1`).
 
 Example: the definition of `console.log`
 
-    console.log = function(d) {
-      process.stdout.write(d + '\n');
+    Console.prototype.log = function() {
+      this._stdout.write(util.format.apply(this, arguments) + '\n');
```

Comment:
oh my, I overlooked that - so it'd make sense to just remove it altogether, wouldn't it?